packages/browser-destinations/destinations/evolv-ai/src/trackEvent/index.ts

```diff
@@ -0,0 +1,42 @@
+import type { BrowserActionDefinition } from '@segment/browser-destination-runtime/types'
+import type { Evolv } from '../types'
+import type { Settings } from '../generated-types'
+import type { Payload } from './generated-types'
+import { sanitiseEventName } from '../utility'
+import { emit } from '../proxy'
+
+// Change from unknown to the partner SDK types
+const action: BrowserActionDefinition<Settings, Evolv, Payload> = {
+  title: 'Track Event',
+  description: `Send Segment track event to Evolv AI`,
+  platform: 'web',
+  defaultSubscription: 'type = "track"',
+  fields: {
+    eventName: {
+      description: 'Name of the event.',
+      label: 'Name',
+      required: true,
+      type: 'string',
+      default: {
+        '@path': '$.event'
+      }
+    },
+    properties: {
+      description: 'JSON object containing additional properties associated with the event.',
```

Comment:
```suggestion
      description: 'Object containing additional properties associated with the event.',
```
Removing JSON just to avoid any confusion considering this will be mapped as an Object within Segment